pwd-service/src/main/java/org/pwd/web/contact/ContactController.java

```diff
@@ -0,0 +1,100 @@
+package org.pwd.web.contact;
+
+import com.google.common.base.Preconditions;
+import org.apache.tomcat.util.codec.binary.Base64;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.*;
+import org.springframework.stereotype.Controller;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.client.RestTemplate;
+
+import static org.springframework.web.bind.annotation.RequestMethod.POST;
+
+/**
+ * @author Sławomir Mikulski
+ */
+@Controller
+@RequestMapping("/kontakt")
+public class ContactController {
+
+    public static final String NAME_LABEL = "Imię i Nazwisko:";
+    public static final String EMAIL_LABEL = "Email:";
+    public static final String MOBILE_PHONE_LABEL = "Numer telefonu:";
+    public static final String SITE_URL_LABEL = "Adres strony:";
+    public static final String MESSAGE_LABEL = "Wiadomość:";
+
+    public static final String SUBJECT = "Zgłoszenie ze strony PWD";
+
+    private final String smtpMail;
+    private final String apiKey;
+    private final String smtpUrl;
+
+    @Autowired
+    public ContactController(@Value("${smtp.mailbox}") String smtpMail,
+                             @Value("${MAILGUN_API_KEY}") String apiKey,
+                             @Value("${MAILGUN_DOMAIN}") String smtpUrl) {
+        Preconditions.checkArgument(!smtpMail.isEmpty());
+        Preconditions.checkArgument(!apiKey.isEmpty());
+        Preconditions.checkArgument(!smtpUrl.isEmpty());
+
+        this.smtpMail = smtpMail;
+
+        if( !apiKey.startsWith("api:"))
```

Comment:
skoro wiemy, ze na heroku tego nie ma, to czemu to ifujemy? po prostu trzeba dokleic